phpunit.xml.dist

```diff
@@ -17,6 +17,7 @@
     convertWarningsToExceptions="true"
     defaultTimeLimit="10"
     enforceTimeLimit="true"
+    failOnRisky="true"
```

Comment:
maybe raise separated PR for this alone, and then we have calm time to review a performance fix separately

Reply:
done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5694